main.yml

```diff
@@ -290,6 +290,31 @@ landing:
   channel: '#flip-mode-squad'
   deployment_repo: https://github.com/RedHatInsights/landing-page-frontend-build
 
+migrations:
+  title: Migrations
+  frontend:
+    sub_apps:
+      - id: migration-analytics
+        default: true
+  top_level: true
+
+migration-analytics:
+  title: Migration Analytics
+  api:
```

Comment:
I tried looking for `migration-analytics` on the `/api` andpoint and it looks like there is no such api, but I could find `/api/xavier` do you mind adding
```yaml
  api:
    ...
    apiName: xavier
    ...
```

Reply:
@karelhala I'll add it no problem. I just wonder why the other branches worked fine without `apiName: xavier`? I mean, our config for `ci-stable, qa-stable, etc` did not include `apiName: xavier`

Reply:
As above

Reply:
It totaly got out of my radar, currently the xavier API is not listed on API docs because there is missing openAPI spec but with this applied and if you'd were to expose openapi.json on `/api/xavier/v1/openapi.json` customers could check your API and use it directly without using app UI.

Reply:
@kruai 
Asa - Sorry for the confusion. Myself and Carlos were updating in parallel. Just to reiterate we won't be making a change to this PR. We must release to prod-stable the same configuration that we have tested in ci-stable and qa-stable.